devtools/gradle/gradle-extension-plugin/pom.xml

```diff
@@ -16,12 +16,30 @@
     <name>Quarkus - Extension Gradle Plugin</name>
     <description>Quarkus - Extension Gradle Plugin</description>
 
+    <properties>
+        <jackson.version>2.12.5</jackson.version>
```

Comment:
Is there a reason you are re-defining the version here? The artifacts that use this version are actually managed in the quarkus-bom. Looks like this could be removed.

Reply:
we don't depend on the bom here that's why we have the version. The same thing is done in the bootstrap project: https://github.com/quarkusio/quarkus/blob/cf20772ed4d6c51964a39afafd3fbf29513f37cc/independent-projects/bootstrap/pom.xml#L46

Reply:
Because that `bootstrap` project is in the `independent-projects` and can't inherit the `build-parent`. The Gradle plugins actually do inherit from the `build-parent` which imports the `quarkus-bom`.

Reply:
right, I removed it.